src/js/structs/UniversePackage.js

```diff
@@ -22,6 +22,10 @@ class UniversePackage extends Item {
     );
   }
 
+  hasKnownIssues() {
```

Comment:
Can we add a unit test for this function?

Reply:
no need to. we have an integration test.